### DISCUSSION TITLE
Opportunity to work on an opensource project. For web developers.

### DISCUSSION THREAD

**Discussion by GrimmReaper1O1** [community/community#134613] [Programming Help]
### Select Topic Area

Question

### Body

I'm relatively new to programming, only been doing it for about a year, but I've been working somewhat full time on a website and I want to take it into the realm of opensource for the use of schools and the under privileged.
So far it has a fair few bugs but the questionnaire is functioning perfectly and is devised from php and javascript. There is little html or css at this point.
The questionnaire has what I coin question sets for each individual question. There are as many questions as the administrators want to enter that revolve each time the page is refreshed via a form button. Each question revolves on it's own accord and the php interpreter gathers the information for the questions with 3 queries to the database on the first run of the page then on each refresh it makes one request to save computational time. All question information is selected in this one query through their id number.
Each class is split up into thirty different subjects dependent on filling them to limit the amount of information in the database table to be searched through.
There are pre-existing administrative functions to fill the questionnaire and the website contains html5 doughnut charts and line graphs to see the results of the tests and track them.
Currently there is more php to be done in terms of the administration aspects in regards to some bugs made by myself after refactoring and I'm going to create search functionality for finding classes in a bio yet to be created as well as the system search functionality to find classes for users in order for teachers to check on their students.
I've made a library for the system as well utilizing php.
The whole thing is built for AZURE or mssql servers but is easily changed to mysql. I'm a little lacking on database knowledge though but I do my best. Please let me know if you want to get in on this project. It is currently private but I could make it public if it is advisable.
As I work alone, I apologize for my lack of indentation in my code, trying to save time. /home/White/Downloads/Questionniare_5.1_needs_bug_fix_bio.zip
[Questionniare_5.1_needs_bug_fix_bio.zip](https://github.com/user-attachments/files/16487674/Questionniare_5.1_needs_bug_fix_bio.zip)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @GrimmReaper1O1 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by GrimmReaper1O1** [reply]
Thank you! I'm new to Github so I really haven't delved into it much as of yet. Sorry for the late reply.
